build.sbt

```diff
@@ -68,8 +68,12 @@ val root = {
       libraryDependencies ++= dep
     )
     .settings(executableScriptName := name.value)
-    .settings(inConfig(Integration)(Defaults.testSettings
-      ++ org.scalafmt.sbt.ScalafmtPlugin.scalafmtConfigSettings :+ (Test / parallelExecution := false)): _*)
+    .settings(
+      inConfig(Integration)(
+        Defaults.testSettings
```

Comment:
👍

Reply:
This change seems like the result of PR is not up to date (I solved some nasty conflicts yesterday, so I do not blame automatic tools like GH for bad visualization of changes :) ). There are no changes in build.sbt.